content/docs/components/serving/kfserving.md

```diff
@@ -1,33 +1,31 @@
 +++
 title = "KFServing"
-description = "Model Serving using KFServing"
+description = "Model serving using KFServing"
 weight = 2
 +++
 
-KFServing enables Serverless inferencing on Kubernetes and provides performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and ONNX to solve production model serving use cases
+KFServing enables serverless inferencing on Kubernetes and provides performant, high abstraction interfaces for common machine learning (ML) frameworks like TensorFlow, XGBoost, scikit-learn, PyTorch, and ONNX to solve production model serving use cases.
 
-KFServing:
+You can use KFServing to do the following:
 
-* Provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML models on arbitrary frameworks.
+* Provide a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML models on arbitrary frameworks.
 
-* Encapsulates the complexity of autoscaling, networking, health checking, and server configuration to bring cutting edge serving features like GPU autoscaling, scale to zero, and canary rollouts to your ML deployments
+* Encapsulate the complexity of autoscaling, networking, health checking, and server configuration to bring cutting edge serving features like GPU autoscaling, scale to zero, and canary roll outs to your ML deployments.
```

Comment:
Actually, "rollouts" in this case should probably be one word rather than two, as it's used as a noun. 

FYI, for a fun way to tell whether a word should be one word or two:
https://ffeathers.wordpress.com/2018/05/20/login-or-log-in-sign-up-or-signup-how-to-tell-when-to-use-one-word-when-two/

Reply:
Oh my, the noun/verb thing. Silly mistake on my behalf! I've reverted the change.

Reply:
:)